v2/paths/recognition/GroupsSearch.js

```diff
@@ -0,0 +1,75 @@
+module.exports = {
+  get: {
+    operationId: 'recognition/groups-search',
+    security: [
+      {
+        Bearer: [],
+      },
+    ],
+    parameters: [
+      {
+        name: 'limit',
+        in: 'query',
+        description: 'Number of maximum objects in the query',
+        type: 'number',
+        required: true,
+      },
+      {
+        name: 'offset',
+        in: 'query',
+        description: '',
```

Comment:
Missing description.

Reply:
Also I updated query to body and implemented GroupSearchRequest